package.json

```diff
@@ -25,15 +25,23 @@
     "yargs": "^3.24.0"
   },
   "dependencies": {
+    "atob": "^2.0.3",
```

Comment:
just make sure that you actually use all of the packages you added, and remove those that you are not using

Reply:
removed "atob" and "btoa"